src/backend/scripts/setup_dev_data.sh

```diff
@@ -37,8 +37,43 @@ ${local_aws} secretsmanager update-secret --secret-id aspen-config --secret-stri
   "S3_external_auspice_bucket": "aspen-external-auspice-data"
 }' || true
 
+echo "Creating SSM Parameter"
```

Comment:
dumb question, why are we creating a new SSM parameter here and not in the terraform code?

Reply:
They are in terraform code, this is for mocking the params in the localstack.